usaspending_api/search/filters/time_period/query_types.py

```diff
@@ -1,13 +1,27 @@
-from .time_period import TimePeriod
+from usaspending_api.search.filters.time_period import AbstractTimePeriod
 
 
-class TransactionSearchTimePeriod(TimePeriod):
-    """A base time period implementation that's designed to suit the time period filters
+class TransactionSearchTimePeriod(AbstractTimePeriod):
+    """A time period implementation that's designed to suit the time period filters
     on transaction search.
+
+    This concrete implementation supports the filter [TransactionSearchTimePeriodObject](#usaspending_api/api_contracts/search_filters.md#transaction-search-time-period-object).
     """
 
-    def __init__(self, default_start_date, default_end_date, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+    def __init__(self, default_start_date: str, default_end_date: str):
+        """Constructor to create TransactionSearchTimePeriod object.
+
+        Parameters
+        ----------
+        default_start_date : str
+            This is the start date value to use when start date is not present in the filter value.
+            See start date in [TransactionSearchTimePeriodObject](#usaspending_api/api_contracts/search_filters.md#transaction-search-time-period-object)
+            for definition.
+        default_end_date : str
+            This is the end date value to use when end date is not present in the filter value.
+            See end date in [TransactionSearchTimePeriodObject](#usaspending_api/api_contracts/search_filters.md#transaction-search-time-period-object)
+            for definition.
```

Comment:
Minor thing on standard for docstrings -- look to the Google Python Style guide: https://google.github.io/styleguide/pyguide.html#383-functions-and-methods

.e.g. `Args` instead of Parameters with lines under

Reply:
Oh gotcha. Ok I'll make that update. I was not aware that was the team standard. I believe I'm using numpy.